content/en/community/_index.html

```diff
@@ -8,7 +8,7 @@
     <main>
         <div class="content">
             <h3>Ensuring Kubernetes works well everywhere and for everyone.</h3>
-            <p>Connect with the Kubernetes community on our <a href="http://slack.k8s.io/">Slack channel</a> or join the
+            <p>Connect with the Kubernetes community on our <a href="http://slack.k8s.io/">Slack channel</a>, <a href="https://discuss.kubernetes.io/">Discussion Board</a>, or join the
```

Comment:
Maybe format these as a bullet list and talk about what each one is best for? Also I don't think "discussion board" needs to be capitalized.